shopware/k8s-meta/1.0/config/packages/operator.yaml

```diff
@@ -0,0 +1,60 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/shopware/shopware/trunk/config-schema.json
+
+parameters:
+    env(K8S_FILESYSTEM_PRIVATE_BUCKET): ""
+    env(K8S_FILESYSTEM_PUBLIC_BUCKET): ""
+    env(K8S_CDN_URL): ""
+    env(K8S_FILESYSTEM_REGION): ""
+    env(K8S_FILESYSTEM_URL: ""
+    env(K8S_CACHE_TYPE): 'cache.adapter.redis_tag_aware'
+    env(K8S_CACHE_URL): "redis://localhost:6379"
+shopware:
+    api:
+        jwt_key:
+            use_app_secret: true
+    admin_worker:
+        enable_admin_worker: false
+        enable_notification_worker: false
+        enable_queue_stats_worker: false
+    deployment:
+        cluster_setup: true
+    cache:
+        tagging:
+            each_snippet: true
+            each_config: false
+            each_theme_config: false
+    increment:
+        user_activity:
+            type: array
+        message_queue:
+            type: array
+    filesystem:
+        private:
+            type: "amazon-s3"
+            visibility: "private"
+            config:
+                bucket: "%env(K8S_FILESYSTEM_PRIVATE_BUCKET)%"
+                region: "%env(K8S_FILESYSTEM_REGION)%"
+                endpoint: "%env(K8S_FILESYSTEM_ENDPOINT)%"
+                use_path_style_endpoint: true
+        public: &public_bucket
+            type: "amazon-s3"
+            url: "%env(K8S_FILESYSTEM_PUBLIC_URL)%"
+            visibility: "public"
+            config:
+                bucket: "%env(K8S_FILESYSTEM_PUBLIC_BUCKET)%"
+                region: "%env(K8S_FILESYSTEM_REGION)%"
+                endpoint: "%env(K8S_FILESYSTEM_ENDPOINT)%"
+                use_path_style_endpoint: true
+        theme: *public_bucket
+        asset: *public_bucket

```

Comment:
In my opinion, theme and assets should be rather built in the container image. Both are application version dependent and ephemeral. Otherwise the CI needs have an access to the S3 bucket to publish the files there. Some cleanup old assets would be necessary too. This makes things complicated.

We workaround it so far by changing the `theme-config` folder location from `/files` to `/var` which isn't a shared volume. It allows to store the theme config in the git repository, use to for build in the CI. `public/bundles` and `public/theme` are also stored in the container.

Reply:
asset yes. Theme I would never recommend building it before in the CI. The hassle is not worth.

You don't need S3 permissions in the CI, you would run `theme:compile` on the actual system, before rotating all active web containers.

Reply:
Right. that's indeed possible since Shopware 6.5 doesn't refer to the theme by sales channel ID anymore. Up to Shopware 6.4, Shopware would overwrite the theme which would cause an issue with the `rolling update` type of deployment.